Core/src/org/sleuthkit/autopsy/report/infrastructure/ReportWizardPortableCaseOptionsVisualPanel.java

```diff
@@ -16,13 +16,16 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.sleuthkit.autopsy.report;
+package org.sleuthkit.autopsy.report.infrastructure;
```

Comment:
See prior comments about portable case UI components.

Reply:
@rcordovano per our discussion, we will leave code as is.